R/inbo_rapport.R

```diff
@@ -15,10 +15,22 @@
 #' @param fig_crop \code{TRUE} to automatically apply the \code{pdfcrop} utility
 #'   (if available) to pdf figures.
 #' @param pandoc_args Additional command line options to pass to pandoc
-#' @param babel What language(s) to use in the document.
+#' @param main_language The main language of the document.
 #' Only used with `style = "Flanders"`.
-#' If you specify multiple languages, the main language is the last one.
-#' Defaults to `"french,dutch,english"`.
+#' Defaults to `"english"`.
+#' Note that `style = "INBO"` and `style = "Vlaanderen"` will always use
+#' `"dutch"` as the main language.
+#' @param other_languages A vector of other languages you want to use within the
+#' document.
+#' Only used with `style = "Flanders"`.
+#' Defaults to `c("dutch", "french")`.
+#' Allowed languages are `"dutch"`, `"french"` and `"english"`.
+#' Note that `style = "INBO"` and `style = "Vlaanderen"` will always use
+#' `c("english", "french")` as the other languages.
+#' Use `\bdutch` before and `\edutch` after the text in Dutch.
+#' Use `\benglish` before and `\eenglish` after the text in English.
+#' Use `\bfrench` before and `\efrench` after the text in French.
+#' You don't need to set this for the main language.
```

Comment:
That's helpful :+1: . I like the approach with the new parameters, it's more straightforward indeed.

I suggest to add (could be in the Details section as well) what a language setting actually impacts while knitting: bibliography (only by main language), hyphenation, other things? It would further clarify why a user must bother about this (also relevant for sections in other languages).

Reply:
I.e. include in the documentation what you referred to [above](https://github.com/inbo/INBOmd/pull/69#discussion_r533982613):

> It affects hyphenation and names of figures, tables, table of content, ...

Reply:
reworked in 724ba59